ta/os_test/os_test.c

```diff
@@ -944,25 +927,21 @@ static void incr_values(size_t bufsize, uint8_t *a, uint8_t *b, uint8_t *c)
 	}
 }
 
+#define TA2TA_BUF_SIZE		(2 * 1024)
 TEE_Result ta_entry_ta2ta_memref(uint32_t param_types, TEE_Param params[4])
 {
 	static const TEE_UUID test_uuid = TA_OS_TEST_UUID;
 	TEE_TASessionHandle sess = TEE_HANDLE_NULL;
-	TEE_Param l_params[4];
-	const size_t bufsize = 2 * 1024;
-	uint8_t in[bufsize];
-	uint8_t inout[bufsize];
-	uint8_t out[bufsize];
+	TEE_Param l_params[4] = { };
+	const size_t bufsize = TA2TA_BUF_SIZE;
```

Comment:
It seems the const variable is unused now.
Any particular reason why you're using a macro instead?

Reply:
Fair enough.